### PR TITLE
#1 update Collect text and remove extra space on title

### LINKED ISSUE #1: Update collect text
- [x] add the word test to the title

### PR DESCRIPTION
**Pull request by samaSiriwardhanage**
Resolves #1

Please add a short description of the work here.

### Acceptance Criteria
- [x] add the word test to the title 
- [ ] 
### Relevant design files
Add a links to the relevant design files or leave as none.
* None

### Testing instructions
Add a list of steps required to test the feature.
1. None

### DoD
For requester to complete:
- [ ] All acceptance criteria are met
- [ ] New logic has been documented
- [ ] New logic has appropriate unit tests
- [ ] Changelog has been updated if necessary
- [ ] Deployment / migration instruction have been updated if required

### CODE REVIEW

File: app/templates/up_next.html [hunk 1]
```diff
@@ -39,7 +39,7 @@
           <div id='indigenous' class="indigenous">
           <img src='/static/indigenous.png'/>
             <!-- <div>This work contains</div>
-            <div class='indigenous_bold'>FIRST PEOPLES CONTENT</div> -->
+            <div class='indigenous_bold'>FIRST PEOPLES CONTENT </div> -->
```
Comment (sighmon) [inline app/templates/up_next.html:42]: Could you please remove the extra space you added here?

**Review by sighmon — CHANGES_REQUESTED** [review]
Hi Sama, this is what's called a code review response.

As you can see, I've requested a change before we merge this code back into your `main` branch.

So would you mind making the change requested, and then let me know when you have?

Cheers, Simon.

File: app/templates/up_next.html (hunk 1 above)
Comment (samaSiriwardhanage) [reply app/templates/up_next.html]: Hi sighmon,
I removed the extra space and push into github. 
Thanks

**Review by sighmon — CHANGES_REQUESTED** [review]
Nice work removing the extra space.

Your code change looks good so far - but when someone taps their Lens to collect the currently playing video, the text will be updated back to `COLLECT`.

To fix this you'll need to search for `COLLECT` in the `app/static/playlist.js` file and change the text there too.

**Hint**: If it appears in more than one place, you might want to use a `constant` to store the text, so you only have to put the text string in the `.js` file once.

**Comment by samaSiriwardhanage** [comment]
Hi Simon, I have updated the playlist.js file. Thanks

**Review by sighmon — APPROVED** [review]
Looks good! The next thing to do is to update the title and description of this Pull Request so that it matches the code change you made.

You might also want to update the title/description of the [Issue](https://github.com/samaSiriwardhanage/playlist-label/issues/1) too.

Then I think this is ready to merge your `add/1-test` branch into `main`.

You'll find the merge button at the bottom of your Pull Request:

merge-button

**Pull request merged**